Makefile

```diff
@@ -1 +1,3 @@
 -include node_modules/marketplace-gulp/Makefile
+
+TEST_URL?='http://localhost:8674'
```

Comment:
I mean you can move that to marketplace-gulp's Makefile so we don't have to configure it for each project. It's more DRY